### PR TITLE
[IA-2464] Un-ignore app tests

### PR DESCRIPTION
**Pull request by rtitle**
\<your comments for this PR go here\>

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by rtitle** [comment]
jenkins retest

**Comment by jdcanas** [comment]
a lot of seemingly unrelated issues https://fc-jenkins.dsp-techops.broadinstitute.org/job/leonardo-fiab-test-runner/31911/console... jenkins retest

**Comment by rtitle** [comment]
gpalloc seems sad.. hopefully will resolve itself soon

**Comment by jdcanas** [comment]
jenkins retest

File: http/src/main/resources/reference.conf [hunk 1]
```diff
@@ -189,7 +189,7 @@ gke {
     ]
     # See https://cloud.google.com/kubernetes-engine/docs/release-notes
     # As of 2020-10-09, 1.16.x is the default but it does not work with the Galaxy chart.
-    version = "1.15.12-gke.4000"
+    version = "1.16.15-gke.6000"
```
Comment (rtitle) [inline http/src/main/resources/reference.conf:192]: Seeing if the default GKE version works 
image

**Comment by rtitle** [comment]
jenkins retest

**Comment by rtitle** [comment]
jenkins retest

**Pull request merged**